File: library/src/main/java/com/mikepenz/aboutlibraries/util/MovementCheck.java
package com.mikepenz.aboutlibraries.util;

import android.text.Spannable;
import android.text.method.LinkMovementMethod;
import android.view.MotionEvent;
import android.widget.TextView;

public class MovementCheck extends LinkMovementMethod {

    @Override
    public boolean onTouchEvent(TextView widget, Spannable buffer, MotionEvent event) {
        try {
            return super.onTouchEvent(widget, buffer, event);
        } catch (Exception ex) {
            return true;
        }
    }

}

File: library/src/main/java/com/mikepenz/aboutlibraries/util/MovementCheck.kt
package com.mikepenz.aboutlibraries.util

import android.text.Spannable
import android.text.method.LinkMovementMethod
import android.view.MotionEvent
import android.widget.TextView


class MovementCheck : LinkMovementMethod() {

    override fun onTouchEvent(widget: TextView, buffer: Spannable, event: MotionEvent): Boolean {
        return try {
            super.onTouchEvent(widget, buffer, event)
        } catch (ex: Exception) {
            true
        }

    }

    private object Holder { val INSTANCE = MovementCheck() }

    companion object {
        val instance: MovementCheck by lazy { Holder.INSTANCE }
    }
}